src/htmlize/patchtml.ml

```diff
@@ -20,10 +19,139 @@ let edit_html ?header ?footer ~head_childs html =
     footer
     "#footer";
   let head = html $ "head" in
-  List.iter (fun (id,child) -> 
-      opt_inserter
-        (iter (fun e -> append_child head e))
-        (Some child)
-        id)
+  List.iter (fun (id,child) ->
+    opt_inserter
+      (iter (fun e -> append_child head e))
+      (Some child)
+      id)
     head_childs;
   to_string html
+
+open Omd
+(* Keeps us from processing url links *)
+let is_url url =
+  let http = String.sub url 0 7 in
+  if String.equal http "http://" then
+    true
+  else
+    let https = String.sub url 0 8 in
+    String.equal https "https://"
+
+(* Keeps us from processing img links *)
+let is_img file =
```

Comment:
why don't you want to process img links ?

Reply:
I thought of an upcoming PR to handle those